department-of-veterans-affairs/caseflow
# Fix Docket Order sorting in Assign Hearings tabs

Resolves #9437 

### Description
Fixed the sorting logic in AssignHearingsTabs to properly sort within each category: CAVC, AOD and normal.

### Acceptance Criteria

### Testing Plan
1. Go to Assign Hearings page
2. Go to Legacy Veterans Waiting tab
3. Confirm Veteran appeals are sorted by docket number within each category.
